spec/instance_topic_fields_spec.rb

```diff
@@ -0,0 +1,70 @@
+RSpec.describe SparqlToSwSolr::InstanceSolrDoc::InstanceTopicFields do
+
+  let(:instance_uri) { 'http://ld4p-test.stanford.edu/1234567890#Instance' }
+  let(:isd) { SparqlToSwSolr::InstanceSolrDoc.new(instance_uri) }
+
+  context 'SPARQL' do
+    let(:sparql_conn) { double('sparql client') }
+    let(:solutions) { RDF::Query::Solutions.new }
+
+    before(:each) do
+      allow(isd).to receive(:sparql).and_return(sparql_conn)
+      expect(sparql_conn).to receive(:query).and_return(solutions)
+    end
+
+    shared_examples 'it_indexes_topics' do
```

Comment:
sadly, the shared_examples probably is not useful here, now that it's clear (see above) that the values of topic_search are not the same as the values of topic_facet.